### DISCUSSION TITLE
Display website name on mobile

### DISCUSSION THREAD

**Discussion by oscarbaruffa** [plausible/analytics#2171] [Anything Else]
When in mobile view, the website name collapses from the view, so you can't tell which site's stats you are viewing. 

Would be great if this didnt happen and the site name was always visible. 

Desktop view = good
![desktop view](https://user-images.githubusercontent.com/16580713/188358163-1ff8a4ac-95a7-499b-81a4-831fbb98559a.png)

Mobile view - site name not there
![mobile view](https://user-images.githubusercontent.com/16580713/188358201-ec1c29ff-ad10-48ca-a156-c31fef03b0a4.png)

**Comment by metmarkosaric** [comment]
thanks for the feedback @oscarbaruffa! did you set up a favicon for your site? currently the generic icon shows up but if we identify a favicon, we would show it which would allow you to see what website it is. we used to show the full domain even on mobile but had to remove it to have features such as filters on mobile.

**Reply by oscarbaruffa** [reply]
Ah ok, I'll add favicons to all sites. Thanks
